### ISSUE TITLE
limits and p_limits not truncating stat_eye()

### ISSUE THREAD

**Issue by maxfarrell** [mjskay/ggdist#227]
I'm plotting some conditional draws and some values are very extreme, leading to the stat_eye plots having very long tailed distributions, and a lot of empty area of the plot.

To reduce the plot area and zoom in in the informative parts of the distributions I'm trying to use "limits" and/or "p_limits" in the stat_eye() function to truncate these, but there is no change in the plot size. 

I can always set scale limits in ggplot manually, but I thought I'd flag this

**Comment by mjskay** [comment]
Yeah, those arguments are intended for analytical distributions, where distributions with non-finite limits require us to provide a way to determine reasonable bounds. Sample-based distributions largely ignore these and take limits from the scale itself, with the expectation you would use `coord_cartesian(xlim = ... )` or `coord_cartesian(ylim = ... )` to set custom limits.
